staging/src/k8s.io/credentialprovider/secrets/secrets.go

```diff
@@ -20,7 +20,7 @@ import (
 	"encoding/json"
 
 	"k8s.io/api/core/v1"
-	"k8s.io/kubernetes/pkg/credentialprovider"
+	"k8s.io/credentialprovider"
```

Comment:
Should move credentialprovider under ?cloud-provider

Reply:
or kubelet?